flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/TableConfigOptions.java

```diff
@@ -51,4 +52,24 @@ private TableConfigOptions() {}
 			.withDescription("The SQL dialect defines how to parse a SQL query. " +
 					"A different SQL dialect may support different SQL grammar. " +
 					"Currently supported dialects are: default and hive");
+
+	@Documentation.TableOption(execMode = Documentation.ExecMode.BATCH_STREAMING)
+	public static final ConfigOption<String> LOCAL_TIME_ZONE = key("table.local-time-zone")
+			.stringType()
+			// special value to decide whether to use ZoneId.systemDefault() in TableConfig.getLocalTimeZone()
+			.defaultValue("System")
```

Comment:
I think it would be better to use "default" as the default value here. Because this is very easy to understand literally and `table.sql-dialect` also uses it.